.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
 - package-ecosystem: cargo
   directory: "/"
   schedule:
-    interval: daily
+    interval: weekly
```

Comment:
Why?

Reply:
There are too many ops related PRs otherwise. This is a change at the rust-vmm org level.